packages/components/src/modifiers/hds-tooltip.ts

```diff
@@ -0,0 +1,222 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// Note: the majority of this code is a porting of the existing tooltip implementation in Cloud UI
+// (which was initially implemented in Structure)
+
+import Modifier from 'ember-modifier';
+import type { ArgsFor } from 'ember-modifier';
+
+import { assert } from '@ember/debug';
+import { registerDestructor } from '@ember/destroyable';
+
+import tippy, { followCursor } from 'tippy.js';
+import type {
+  Props as TippyProps,
+  Instance as TippyInstance,
+  HideAll,
+  MultipleTargets,
```

Comment:
Do we want to establish a new best practice here? I differentiated here between instance purposefully so it wouldn't get mistaken for the class instance which gets passed to the cleanup function.  I was studying your code in hds-anchored-position.ts and noticed none of your [@floating-ui type imports were prefixed](https://github.com/hashicorp/design-system/blob/e4cf01addb3c650a2e1871e575db2d7bf27e7b3e/packages/components/src/modifiers/hds-anchored-position.ts#L32) which is why I left the others un-prefixed.  Do you want me to create a PR so that your code matches in consistency and possibly add documentation somewhere so other devs know what is expected ahead of time?

Reply:
@alex-ju What do you think, should we add a section to the [TypeScript Migration Guide Wiki ](https://github.com/hashicorp/design-system/blob/main/wiki/TypeScript-Migration.md)and establish a best practice for how we import/consume third-party types.